src/script/auth/component/AccountForm.tsx

```diff
@@ -106,27 +106,26 @@ const AccountForm = ({account, ...props}: Props & ConnectedProps & DispatchProps
       await props.doSendActivationCode(registrationData.email);
       return props.onSubmit(event);
     } catch (error) {
-      if (error && (error instanceof ValidationError || error instanceof BackendError) && error.label) {
-        switch (error.label) {
+      const label = (error as BackendError)?.label;
```

Comment:
error instanceof ValidationError why this check is removed? @atomrc

Reply:
It created runtime errors, because we are receiving more than just Validation and Backend errors, we would need to find out **all** the different classes that go there, but for now I'd like to go on the safe side and handle anything that has a `label`